pages/1.10/cli/enterprise-cli/index.md

```diff
@@ -579,7 +579,7 @@ Usage: dcos security secrets create [OPTIONS] PATH
 Options:
   -s, --store-id TEXT        Secrets backend to use.
   -v, --value TEXT           Value of the secret.
-  -f, --value-file FILENAME  Treat contents of the file as value of the secret.
+  -f, --file FILENAME  Treat contents of the file as value of the secret.
```

Comment:
We need to move `Treat` word to match the indentation of the remaining help text.